packages/client/src/__tests__/integration/happy/transaction/test.ts

```diff
@@ -18,8 +18,8 @@ function clean(array: any[]) {
   })
 }
 
-beforeAll(() => {
-  fs.copyFileSync(path.join(__dirname, 'dev.db'), path.join(__dirname, 'dev2.db'))
+beforeAll(async () => {
+  await fs.promises.copyFile(path.join(__dirname, 'dev.db'), path.join(__dirname, 'dev2.db'))
```

Comment:
Probably doesn't matter in this case. Technically it'll even be kinda slower because of the overhead of creating a new task on the thread pool and then just waiting for it without doing anything else on the main thread (since there shouldn't be anything else running concurrently with the `beforeAll` hook in one Jest worker), but the difference should be negligible.